ui_framework/components/index.js

```diff
@@ -4,9 +4,15 @@ export {
   KuiButtonIcon,
   KuiLinkButton,
   KuiSubmitButton,
-} from './button';
+} from './button/index';
```

Comment:
Is there a reason you explicitly have to include `index` here? This doesn't seem to be related to this PR at all...